docs/Installation-Windows.md

```diff
@@ -0,0 +1,168 @@
+    <img src="images/cudnn_zip_files.PNG" 
+        alt="cuDNN zip files" 
+        width="500" border="10" />
+</p>
+
+Copy these three folders into the CUDA toolkit directory.  In this guide, the CUDA toolkit directory is located at `C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.1`
+
+<p align="center">
+    <img src="images/cuda_toolkit_directory.PNG" 
+        alt="cuda toolkit directory" 
+        width="500" border="10" />
+</p>
+
+## Step 3: Set Environment Variables
+You will need to add one environment variable and two path variables.
+
+To set the environment variable, type `environment variables` in the search bar (this can be reached by hitting the Windows key or the bottom left Windows button).  You should see an option called __Edit the system environment variables__.  
+
+<p align="center">
+    <img src="images/edit_env_var.png" 
+        alt="edit env variables" 
+        width="250" border="10" />
+</p>
+
+From here, click the __Environment Variables__ button.  Click __New__ to add a new system variable _(make sure you do this under __System variables__ and not User variables_.  
+
+<p align="center">
+    <img src="images/new_system_variable.PNG" 
+        alt="new system variable" 
+        width="500" border="10" />
+</p>
+
+For __Variable Name__, enter `CUDA_HOME`.  For the variable value, put the directory location for the CUDA toolkit.  In this guide, the directory location is `C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.1`.  Press __OK__ once.
+
+<p align="center">
+    <img src="images/system_variable_name_value.PNG" 
+        alt="system variable names and values" 
+        width="500" border="10" />
+</p>
+
+To set the two path variables, inside the same __Enviornment Variables__ window and under the second box called __System Variables__, find a variable called `PATH` and click __Edit__.  You will add  two directories to the list.  For this guide, the two entries would look like:
+
+    C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.1\lib\x64
+    C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.1\extras\CUPTI\libx64
+
+Make sure to replace the relevant directory location with the one you have installed.  _Please note that case sensitivity matters_.
+
+<p align="center">
+    <img src="images/path_variables.PNG" 
+        alt="Path variables" 
+        width="500" border="10" />
+</p>
+
+## Step 4: Install Python via Anaconda
+<a href="https://www.anaconda.com/download/#windows" target="_blank">Download</a> and install Anaconda for Windows.  By using Anaconda, you can use manage separate environments for different distributions of Python.  We **strongly** recommend using Python 3 as we do not guarantee supporting Python 2 in future releases.  In this guide, we are using Python version 3.6 and Anaconda version 5.1 ([64-bit](https://repo.continuum.io/archive/Anaconda3-5.1.0-Windows-x86_64.exe) or [32-bit](https://repo.continuum.io/archive/Anaconda3-5.1.0-Windows-x86.exe) direct links).
```

Comment:
Ensure that _blank isn't impacting markdown. I use this: https://github.com/joeyespo/grip